multi-agent/lib/mas/resources/field_encryption.py

```
@@ -0,0 +1,199 @@
+            read_only_hint = hints.get("read_only", {})
+            if read_only_hint.get("read_only") is False:
+                configurable.add(field_name)
+            if "secret" in hints:
+                sensitive.add(field_name)
+        return configurable, sensitive
+
+    def find_missing_conditionally_required_secrets(
+        self,
+        category: str,
+        type_key: str,
+        resolved_config: Any,
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `resolved_config`

(ANN401)